src/app/components/SubjectHeading/AdditionalSubjectHeadingsButton.jsx

```diff
@@ -65,7 +65,7 @@ class AdditionalSubjectHeadingsButton extends React.Component {
 
     return (
       <tr
-        className="subjectHeadingRow nestedSubjectHeading"
+        className={`subjectHeadingRow nestedSubjectHeading ${previous || noEllipse ? '' : 'ellipse'}`}
```

Comment:
I think the `nestedSubjectHeading` classname is not necessary for this component. Mind removing it as part of this PR?

Reply:
Done